core/src/util/usLDAPExpr.cpp

```diff
@@ -0,0 +1,852 @@
+/*=============================================================================
```

Comment:
I searched online, and found that it seems not possible with github UI. 
Summary of changes:
- use usProperties class instead of usServicePropertiesImpl class.